client/src/www/views/servers_view/server_list_item/server_card/index.ts

```diff
@@ -243,6 +248,14 @@ const getSharedComponents = (element: ServerListItemElement & LitElement) => {
           </md-text-button>
         </footer>
       `,
+      renameDialog: html`<server-rename-dialog
+        .open=${element.isRenameDialogOpen}
+        .localize=${localize}
+        serverId=${server.id}
```

Comment:
Why are `serverId` and `serverName` attributes expressions? Shouldn't they also be [property expressions](https://lit.dev/docs/templates/expressions/#property-expressions)?